CMakeLists.txt

```diff
@@ -26,9 +37,21 @@ file(
 	src/HexEditorCtrl/wxHexCtrl/wxHexCtrl.h
 )
 
+if (WIN32)
+	list(APPEND hex_editor_ctrl_sources
+		resources/resource.rc
+		windrv.cpp
+		windrv.h
+	)
+endif()
+
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/bin)
-add_executable(wxHexEditor ${hex_sources} ${hex_editor_ctrl_sources})
+add_executable(wxHexEditor ${hex_editor_ctrl_sources})
 target_include_directories(wxHexEditor PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
 target_link_libraries(
-	wxHexEditor PRIVATE mhash udis ${wxWidgets_LIBRARIES} uxtheme oleacc psapi ws2_32
+	wxHexEditor PRIVATE mhash udis ${wxWidgets_LIBRARIES}
 )
+
+if (WIN32)
+	target_link_libraries(wxHexEditor PRIVATE mhash udis ${wxWidgets_LIBRARIES} uxtheme oleacc psapi ws2_32)
```

Comment:
I don't think you need to include the libraries that were already added -- for windows, you can just append the windows-specific libraries.

Per the cmake documentation:
>  target_link_libraries: Repeated calls for the same <target> append items in the order called.